TESTS/netsocket/README.md

```diff
@@ -1068,527 +1088,513 @@ Test whether timeouts are obeyed in TCPSockets.
 **Preconditions:**
 
-1.  Network interface and stack are initialized.
-2.  Network connection is up.
+1. Network interface and stack are initialized.
+1. Network connection is up.
 
 **Test steps:**
 
-1.  Create TLSSocket.
-2.  Call `TLSSocket::open()`.
-3.  Call `TLSSocket::connect("os.mbed.com", 2009)`.
-4.  Call `TLSSocket::close()`.
+1. Create TLSSocket.
+1. Call `TLSSocket::open()`.
+1. Call `TLSSocket::connect("os.mbed.com", 2009)`.
+1. Call `TLSSocket::close()`.
 
 **Expected result:**
 
-TLSSocket::connect must return an error
+`TLSSocket::connect` must return an error.
 
 ### TLSSOCKET_SEND_CLOSED
 
 **Description:**
 
-Make a HTTP request to a closed socket.
+Make an HTTP request to a closed socket.
 
 **Preconditions:**
 
-1.  Network interface and stack are initialized.
-2.  Network connection is up.
+1. Network interface and stack are initialized.
+1. Network connection is up.
 
 **Test steps:**
 
-1.  Create TLSSocket.
-2.  Call `TLSSocket::open()`.
-3.  Call `TLSSocket::connect("echo.mbedcloudtesting.com", 2007)`.
-4.  Call `TLSSocket::close()`.
-5.  Call `TLSSocket::send("12345", 5)`.
+1. Create TLSSocket.
+1. Call `TLSSocket::open()`.
+1. Call `TLSSocket::connect("echo.mbedcloudtesting.com", 2007)`.
+1. Call `TLSSocket::close()`.
+1. Call `TLSSocket::send("12345", 5)`.
 
 **Expected result:**
 
-TLSSocket::send must return an error
+`TLSSocket::send` must return an error.
 
 ### TLSSOCKET_SEND_REPEAT
 
-**Description:** Run SOCKET_SEND_REPEAT for TLSSOCKET by using port number 2009.
+**Description:** Run `SOCKET_SEND_REPEATÂ` for TLSSOCKET by using port number 2009.
 
 ### TLSSOCKET_SEND_TIMEOUT
 
-**Description:** Run SOCKET_SEND_TIMEOUT for TLSSOCKET by using port number 2009.
+**Description:** Run `SOCKET_SEND_TIMEOUTÂ` for TLSSOCKET by using port number 2009.
 
 ### TLSSOCKET_SEND_UNCONNECTED
 
 **Description:** 
 
-Make a HTTP request to an unconnected socket.
+Make an HTTP request to an unconnected socket.
 
 **Preconditions:**
 
-1.  Network interface and stack are initialized.
-2.  Network connection is up.
+1. Network interface and stack are initialized.
+1. Network connection is up.
 
 **Test steps:**
 
-1.  Create TLSSocket.
-2.  Call `TLSSocket::open()`.
-3.  Call `TLSSocket::send("12345", 5)`.
-4.  Call `TLSSocket::connect("echo.mbedcloudtesting.com", 2007)`.
+1. Create TLSSocket.
+1. Call `TLSSocket::open()`.
+1. Call `TLSSocket::send("12345", 5)`.
+1. Call `TLSSocket::connect("echo.mbedcloudtesting.com", 2007)`.
 
 **Expected result:**
 
-TLSSocket::send must return an error
+`TLSSocket::send` must return an error.
 
 ### TLSSOCKET_ECHOTEST
 
-**Description:** Run SOCKET_ECHOTEST for TLSSOCKET by using port number 2007.
+**Description:** Run `SOCKET_ECHOTESTÂ` for TLSSOCKET by using port number 2007.
 
 ### TLSSOCKET_ECHOTEST_NONBLOCK
 
-**Description:** Run SOCKET_ECHOTEST_NONBLOCK for TLSSOCKET by using port number 2007.
+**Description:** Run `SOCKET_ECHOTEST_NONBLOCKÂ` for TLSSOCKET by using port number 2007.
```

Comment:
It looks like there's some weird formatting going on. Do we know what this symbol should be, or should we delete it?

Reply:
Sorry but i don't know that.

Reply:
Similar to above, the last char should be deleted

Reply:
Yes, you're right. I've removet it.